packages/core/src/codewhisperer/service/inlineCompletionService.ts

```diff
@@ -248,6 +248,7 @@ export class CodeWhispererStatusBar {
             }
             case 'notConnected':
                 statusBar.text = codicon` ${getIcon('vscode-chrome-close')} ${title}`
+                statusBar.backgroundColor = new vscode.ThemeColor('statusBarItem.errorBackground')
```

Comment:
I'm assuming `errorBackground` is red. Do we really want to make it red when they haven't added a connection yet? I don't think we do this for the Toolkit extension, only showing red once expired.

Reply:
Yeah this is intended